### PR TITLE
chore: Error Messages C4 fixes and normalization (SC-2036)

### PR DESCRIPTION
**Pull request by deluca-mike**
# Description

Rectify error-messsage-related issues raised by C4 and and normalize error messages code-wide.

Clubhouse: https://app.clubhouse.io/maplefinance/story/2036/maple-core-c4-error-messages

Slight error message and code refactor needed to  get back under contract size limits after handling original fixes.

# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [x] Has all documentation been updated?

# Changelog

- C4 error message issues 62, 63, 65 (see commit description for links)
- Standardized require error messages as "CONTRACT_ABBREVIATION:MESSAGE"
- Standardized all access error messages as "NOT_X[_OR_Y]"
- Added missing error messages for SafeMath contracts
- Fixed error message typo in LiquidityLocker contract
- Normalized all "LIQUIDITY" to "LIQ" in error messages
- Differentiated "NOT_ALLOWED" error messages
- Rephrased and/or abbreviated some error messages (i.e. _GOV_ernor, _O_ut _O_f _B_ounds, etc)
- Removed redundant bytecode in PoolLib, BasicFDT, ExtendedFDT, LoanFactory, MapleGlobals, Pool)
- Standardized error message abbreviations for common roles (_L_oan, _P_ool, _F_actory, etc)

### CODE REVIEW

**Comment by lucas-manuel** [comment]
@xhad would these error message abbreviations cause any issues for the offchain team?

**Review by satyamakgec — DISMISSED** [review]

File: contracts/LoanFactory.sol [hunk 1]
```diff
@@ -102,12 +102,13 @@ contract LoanFactory is Pausable {
         );
 
         // Update LoanFactory identification mappings
-        loans[loansCreated]   = address(loan);
-        isLoan[address(loan)] = true;
+        loanAddress = address(loan);
+        loans[loansCreated]   = loanAddress;
```
Comment (lucas-manuel) [inline contracts/LoanFactory.sol:106]: `=` alignment

File: contracts/library/LoanLib.sol [hunk 2]
```diff
@@ -128,8 +128,8 @@ library LoanLib {
         @param globals Instance of the `MapleGlobals` contract.
      */
     function reclaimERC20(address token, address liquidityAsset, IGlobals globals) external {
-        require(msg.sender == globals.governor(), "Loan:UNAUTHORIZED");
-        require(token != liquidityAsset && token != address(0), "Loan:INVALID_TOKEN");
+        require(msg.sender == globals.governor(), "L:NOT_GOV");
+        require(token != liquidityAsset && token != address(0), "L:INVALID_TOKEN");
```
Comment (lucas-manuel) [inline contracts/library/LoanLib.sol:132]: Lets align these error messages

**Review by lucas-manuel — CHANGES_REQUESTED** [review]
Small changes

**Review by lucas-manuel — DISMISSED** [review]
LGTM, gonna let CI run.

**Review by lucas-manuel — DISMISSED** [review]

File: contracts/Pool.sol [hunk 3]
```diff
@@ -232,7 +233,7 @@ contract Pool is PoolFDT {
 
         emit Claim(loan, interestClaim, principalClaim, claimInfo[3], stakeLockerPortion, poolDelegatePortion);
 
-        return claimInfo;
+        // TODO: Discuss with offchain team about requirements for return
```
Comment (lucas-manuel) [inline contracts/Pool.sol:236]: This can be taken out now that we've merged the TODOs PR

**Pull request merged**